cms/tests/test_page.py

```diff
@@ -1138,6 +1138,55 @@ def test_page_preview_persists(self):
             self.assertContains(resp, public_text)
             self.assertNotContains(resp, draft_text)
 
+    def test_translated_subpage_title_path_regeneration(self):
+        """
+        When a child page is created with multiple translations before parent translation,
+        child title translation path should be regenerated to take into account parent path.
+        This test enforces the issues found in: https://github.com/django-cms/django-cms/issues/6622,
```

Comment:
Can you please add a line break here. You can accept or maybe reject this suggestion using github. :-)

```suggestion

        This test enforces the issues found in: https://github.com/django-cms/django-cms/issues/6622,
```